### ISSUE TITLE
Can't add User columns

### ISSUE THREAD

**Issue by Krajiyah** [ParsePlatform/parse-dashboard#125]
I am running version 2.1.4 of Parse Server via Heroku. For my Parse Dashboard I have that running locally. I have just cloned repo from scratch. I am able to add new classes and edit columns for those fine......However......for the User class, I cant add any columns.

Here is a screenshot including stack trace for after I fill in type and name of the column dialog popup:

![asd](https://cloud.githubusercontent.com/assets/10042218/13687102/38669322-e6ce-11e5-896a-9b91ccc7cc7a.png)

**Comment by drew-gross** [comment]
Could you tell me why type of column you were trying to add, with what name, and show the response received from the server?

**Comment by Krajiyah** [comment]
I tried to add a String type with column name "test" and the second red console error in the screenshot is the result I got after I clicked "add column"

**Comment by drew-gross** [comment]
If you look in the network tab of the debugger, it will show you all the requests that have been made since the debugger was opened. Click on the request that failed (to `.../schemas/_User`) and show me the response from the server (preview tab).

**Comment by peterdotjs** [comment]
@Krajiyah as a note if you run `npm run dev` the debug console will be more useful than the minified bundled resource.

**Comment by Krajiyah** [comment]
@drew-gross Here is the response (its wierd because the name of the field was not username is was test): 
```
{code: 136, error: "field username cannot be added"}
code: 136
error: "field username cannot be added"
```

@peterdotjs Here is a screenshot of debug console after running npm dev run instead of npm run dashboard

![asd](https://cloud.githubusercontent.com/assets/10042218/13689574/b2ec0c3e-e6de-11e5-912d-481f39698778.png)

**Comment by peterdotjs** [comment]
@Krajiyah this was fixed on parse server version `2.1.5`. Once you update it'll be fixed.

**Comment by alexmiller00** [comment]
When I initialize the server via command line ssh (Ubuntu 14.04) and I go to my parse server https://server_IP:4040 I get an error message telling me to create a user to access remotely how do I add a user from the command line so that I can login to my parse server??
